example/exporter/kube-state-metrics/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/scrape: 'true'
+  name: kube-state-metrics
+  labels:
+    app: kube-state-metrics
+spec:
+  ports:
+  - name: kube-state-metrics
```

Comment:
This can actually be chosen to be reusable similar to [here](https://github.com/brancz/kube-prometheus/blob/f3973972d95abcc8cd5411dcd17cf5645f55460f/manifests/exporters/kube-state-metrics-svc.yaml) in combination with a `ServiceMonitor` like [this](https://github.com/coreos/kube-prometheus/pull/33/files#diff-88a7a3ebbd1605371aae973804dff0c4). Then we just need 2-3 `ServiceMonitor`s to monitor the entire Kubernetes stack.